src/test/java/CreditCardValidateTest.java

```diff
@@ -0,0 +1,52 @@
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.*;
+
+class CreditCardValidatorTest {
```

Comment:
I see no validation of validation mechanism: exactly set of errors.

Reply:
Fixed here: 8741778